02-props-state-component-architecture/props/pokedex/src/Pokedex.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import Pokecard from "./Pokecard";
+import "./Pokedex.css";
+
+export default class Pokedex extends Component {
+	render() {
+		const pokemon = this.props.pokemon.map(poke => {
+			return (
+				<Pokecard
+					key={poke.id}
+					name={poke.name}
```

Comment:
tip: since the prop names are the same as the keys of poke, with the exception of `key`, you can use object spread for shorthand:

```jsx
const { id, ...rest } = poke;
<Pokecard
  key={poke.id}
  {...rest}
 />